### ISSUE TITLE
Trouble with enumerated imports

### ISSUE THREAD

**Issue by danslapman** [NeQuissimus/sort-imports#5]
Affected version: 0.2.0

Example:

```scala
object Foo {
  import A._, B._, C._
}
```

Expected behavior: code remains unchanged

Actual behavior:

```scala
object Foo {
  import A._, 
B._, 
C._
}
```

**Comment by NeQuissimus** [comment]
I believe I fixed this, will release `0.2.1`, so you can try it out. Let me know how it goes.

**Comment by danslapman** [comment]
Works well in all my cases, thank You!
